tools-v2/README.md

```diff
@@ -55,6 +55,7 @@ A tool for CurveFS & CurveBs.
     - [status](#status-1)
       - [staus etcd](#staus-etcd)
       - [staus mds](#staus-mds)
+      - [staus client](#staus-client)
```

Comment:
add follow in #comparison-of-old-and-new-commands/#curve-bs

```markdown
| client-status| curve bs status client|
```

Reply:
done